From 66193976abfc22190f0042f864097e47720ca897 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:15:39 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit eecf781dda34a5f6d52a40183b89d5e2693a802f
Merge: 761e746fc 992d47a23
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 20 23:45:36 2018 +0000

    Snap for 4905103 from 992d47a237b1e2c8ca5d301397a0525f17252497 to pi-dr1-release

    Change-Id: I35f4c07946d149ff8a19ce669bbcf6ac36a666b7

commit 992d47a237b1e2c8ca5d301397a0525f17252497
Merge: a824eac6e b1d8302e3
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Fri Jul 20 18:36:38 2018 +0000

    Merge "Import translations. DO NOT MERGE" into pi-dev

commit a824eac6e2d366d6a653e97157ef7372147ce2d3
Merge: 1754bcd41 315da3188
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Fri Jul 20 18:14:33 2018 +0000

    Merge "Import translations. DO NOT MERGE" into pi-dev

commit 1754bcd410475bff362c51bf2ecac32824e0a08a
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 19 21:21:33 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I0a07d63a0916cf33bbebcf68bd58bef0dfb91826

commit 315da3188282c24564384c488485ccf15f847b6a
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 19 21:20:48 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ifffe3a45508e97013f768a0f7949cb2f8571acee

commit b1d8302e3f5972c271008b1bc701b84c1215668f
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 19 21:18:51 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Icc6eca523dd0cad7c8087aa0ede4edbd7ac9d748

commit 761e746fcc0f730a9d6b24420a93d880e9d5ebf1
Merge: f49ea063b 900a59f17
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:00:32 2018 +0000

    Snap for 4859140 from 900a59f177b5b048bc0bf7883495e4a2b781f758 to pi-dr1-release

    Change-Id: If5972641e5b96d6f4f7a954b3b9b53fc337f366d

commit 900a59f177b5b048bc0bf7883495e4a2b781f758
Merge: 79f43db51 e0a1f83d8
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Fri Jun 22 00:01:23 2018 +0000

    Merge "Import translations. DO NOT MERGE" into pi-dev

commit 79f43db51339dbd269a4f1cfcb94c5d2a11e3be9
Merge: 3fc7472bc d25ca81e5
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Thu Jun 21 22:59:59 2018 +0000

    Merge "Import translations. DO NOT MERGE" into pi-dev

commit 3fc7472bcfbcfd90d4758d1aec38de6c5044a4fa
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 12:18:33 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ifdefd11713addfc8ffedee5f65b9c842cbe3a435

commit e0a1f83d8bda5b80545e5ce9078b6d1fc0befe18
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 12:17:47 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I3a99b392b7d303bac6c0e702029642b75419a6cb

commit d25ca81e5d1329e13446aee415a2754ccc04507f
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 12:15:36 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I8984bc6cf378f0efcbcb0e7a1d601614045c6d10

commit f49ea063b6a529e88a8b300adbf18d0279a9b902
Merge: 6f52cfc82 5223e5571
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:03:28 2018 +0000

    Snap for 4820872 from 5223e55714f9a1e605c51ba577907314655dc436 to pi-dr1-release

    Change-Id: I9f4423c155c3cd731f9d612f40b3ede63465b303

Change-Id: I9152f66ea54f4073bd557b55908d4c4840ae893d
---
 java/res/values-bn/strings-talkback-descriptions.xml | 2 +-
 java/res/values-cs/strings-emoji-descriptions.xml    | 2 +-
 java/res/values-es/strings-talkback-descriptions.xml | 2 +-
 java/res/values-ne/strings-talkback-descriptions.xml | 2 +-
 java/res/values-ne/strings.xml                       | 2 +-
 java/res/values-ru/strings-emoji-descriptions.xml    | 4 ++--
 java/res/values-ru/strings.xml                       | 2 +-
 java/res/values-sw/strings.xml                       | 6 +++---
 java/res/values-zh-rHK/strings.xml                   | 4 ++--
 9 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/java/res/values-bn/strings-talkback-descriptions.xml b/java/res/values-bn/strings-talkback-descriptions.xml
index feb211d60..ff7b0d8a0 100644
--- a/java/res/values-bn/strings-talkback-descriptions.xml
+++ b/java/res/values-bn/strings-talkback-descriptions.xml
@@ -76,7 +76,7 @@
     <string name="spoken_descrption_emoji_category_eight_animals_nature" msgid="5844950234883716704">"জীবজন্তু ও প্রকৃতি"</string>
     <string name="spoken_descrption_emoji_category_eight_food_drink" msgid="6196944764485349650">"খাদ্য ও পানীয়"</string>
     <string name="spoken_descrption_emoji_category_eight_travel_places" msgid="3834085499381434611">"ভ্রমণ ও স্থানগুলি"</string>
-    <string name="spoken_descrption_emoji_category_eight_activity" msgid="4795281669042975993">"কার্যকলাপ"</string>
+    <string name="spoken_descrption_emoji_category_eight_activity" msgid="4795281669042975993">"অ্যাক্টিভিটি"</string>
     <string name="spoken_descrption_emoji_category_emoticons" msgid="456737544787823539">"ইমোটিকনগুলি"</string>
     <string name="spoken_description_upper_case" msgid="4904835255229433916">"বড় হাতের <xliff:g id="LOWER_LETTER">%s</xliff:g>"</string>
     <string name="spoken_letter_0049" msgid="4743162182646977944">"বড় হাতের I"</string>
diff --git a/java/res/values-cs/strings-emoji-descriptions.xml b/java/res/values-cs/strings-emoji-descriptions.xml
index 988abd12d..db072c415 100644
--- a/java/res/values-cs/strings-emoji-descriptions.xml
+++ b/java/res/values-cs/strings-emoji-descriptions.xml
@@ -1100,7 +1100,7 @@
     <string name="spoken_emoji_1F685" msgid="1237059817190832730">"Vysokorychlostní vlak s kulatou přídí"</string>
     <string name="spoken_emoji_1F686" msgid="3525197227223620343">"Vlak"</string>
     <string name="spoken_emoji_1F687" msgid="5110143437960392837">"Metro"</string>
-    <string name="spoken_emoji_1F688" msgid="4702085029871797965">"Železnice"</string>
+    <string name="spoken_emoji_1F688" msgid="4702085029871797965">"Městská dráha"</string>
     <string name="spoken_emoji_1F689" msgid="2375851019798817094">"Stanice"</string>
     <string name="spoken_emoji_1F68A" msgid="6368370859718717198">"Tramvaj"</string>
     <string name="spoken_emoji_1F68B" msgid="2920160427117436633">"Vagon tramvaje"</string>
diff --git a/java/res/values-es/strings-talkback-descriptions.xml b/java/res/values-es/strings-talkback-descriptions.xml
index 139b637f9..4f79193d0 100644
--- a/java/res/values-es/strings-talkback-descriptions.xml
+++ b/java/res/values-es/strings-talkback-descriptions.xml
@@ -74,7 +74,7 @@
     <string name="spoken_descrption_emoji_category_flags" msgid="5971573825866381472">"Banderas"</string>
     <string name="spoken_descrption_emoji_category_eight_smiley_people" msgid="5682663819532433464">"Sonrisas y personas"</string>
     <string name="spoken_descrption_emoji_category_eight_animals_nature" msgid="5844950234883716704">"Animales y naturaleza"</string>
-    <string name="spoken_descrption_emoji_category_eight_food_drink" msgid="6196944764485349650">"Alimentación y bebidas"</string>
+    <string name="spoken_descrption_emoji_category_eight_food_drink" msgid="6196944764485349650">"Comer y beber"</string>
     <string name="spoken_descrption_emoji_category_eight_travel_places" msgid="3834085499381434611">"Viajes y sitios"</string>
     <string name="spoken_descrption_emoji_category_eight_activity" msgid="4795281669042975993">"Actividad"</string>
     <string name="spoken_descrption_emoji_category_emoticons" msgid="456737544787823539">"Emoticonos"</string>
diff --git a/java/res/values-ne/strings-talkback-descriptions.xml b/java/res/values-ne/strings-talkback-descriptions.xml
index 4a8438693..5d4d1315b 100644
--- a/java/res/values-ne/strings-talkback-descriptions.xml
+++ b/java/res/values-ne/strings-talkback-descriptions.xml
@@ -72,7 +72,7 @@
     <string name="spoken_descrption_emoji_category_places" msgid="1163315840948545317">"स्थानहरू"</string>
     <string name="spoken_descrption_emoji_category_symbols" msgid="474680659024880601">"प्रतिकहरू"</string>
     <string name="spoken_descrption_emoji_category_flags" msgid="5971573825866381472">"झण्डाहरू"</string>
-    <string name="spoken_descrption_emoji_category_eight_smiley_people" msgid="5682663819532433464">"स्माइली र व्यक्तिहरू"</string>
+    <string name="spoken_descrption_emoji_category_eight_smiley_people" msgid="5682663819532433464">"स्माइली र मान्छेहरू"</string>
     <string name="spoken_descrption_emoji_category_eight_animals_nature" msgid="5844950234883716704">"पशुहरू र प्रकृति"</string>
     <string name="spoken_descrption_emoji_category_eight_food_drink" msgid="6196944764485349650">"खाद्य र पेय"</string>
     <string name="spoken_descrption_emoji_category_eight_travel_places" msgid="3834085499381434611">"यात्रा र स्थानहरू"</string>
diff --git a/java/res/values-ne/strings.xml b/java/res/values-ne/strings.xml
index 323536087..2a7055317 100644
--- a/java/res/values-ne/strings.xml
+++ b/java/res/values-ne/strings.xml
@@ -153,7 +153,7 @@
     <string name="setup_step2_instruction" msgid="9141481964870023336">"त्यसपछि, \"<xliff:g id="APPLICATION_NAME">%s</xliff:g>\" लाई तपाईँको सक्रिय पाठ इनपुट विधिका रूपमा चयन गर्नुहोस्।"</string>
     <string name="setup_step2_action" msgid="1660330307159824337">"इनपुट विधि स्विच गर्नुहोस्"</string>
     <string name="setup_step3_title" msgid="3154757183631490281">"बधाई छ, तपाईँले सेट पुरा गर्नुभयो!"</string>
-    <string name="setup_step3_instruction" msgid="8025981829605426000">"अब तपाईं <xliff:g id="APPLICATION_NAME">%s</xliff:g>का साथ तपाईंका सम्पूर्ण मनपर्ने अनुप्रयोगहरू टाइप गर्न सक्नुहुन्छ।"</string>
+    <string name="setup_step3_instruction" msgid="8025981829605426000">"अब तपाईं <xliff:g id="APPLICATION_NAME">%s</xliff:g>का साथ तपाईंका सम्पूर्ण मन पर्ने अनुप्रयोगहरू टाइप गर्न सक्नुहुन्छ।"</string>
     <string name="setup_step3_action" msgid="600879797256942259">"थप भाषाहरू कन्फिगर गर्नुहोस्"</string>
     <string name="setup_finish_action" msgid="276559243409465389">"समाप्त भयो"</string>
     <string name="show_setup_wizard_icon" msgid="5008028590593710830">"अनुप्रयोग आइकन देखाउनुहोस्"</string>
diff --git a/java/res/values-ru/strings-emoji-descriptions.xml b/java/res/values-ru/strings-emoji-descriptions.xml
index a133412ff..1c6f4e07a 100644
--- a/java/res/values-ru/strings-emoji-descriptions.xml
+++ b/java/res/values-ru/strings-emoji-descriptions.xml
@@ -249,7 +249,7 @@
     <string name="spoken_emoji_1F194" msgid="4903128609556175887">"Квадратный значок идентификатора."</string>
     <string name="spoken_emoji_1F195" msgid="1433142500411060924">"Квадратный значок Новый."</string>
     <string name="spoken_emoji_1F196" msgid="8825160701159634202">"Квадратный значок Нехорошо."</string>
-    <string name="spoken_emoji_1F197" msgid="7841079241554176535">"Квадратный значок О\'кей."</string>
+    <string name="spoken_emoji_1F197" msgid="7841079241554176535">"Квадратный значок Окей."</string>
     <string name="spoken_emoji_1F198" msgid="7020298909426960622">"Квадратный значок СОС."</string>
     <string name="spoken_emoji_1F199" msgid="5971252667136235630">"Квадратный значок обновления с восклицательным знаком."</string>
     <string name="spoken_emoji_1F19A" msgid="4557270135899843959">"Квадратный значок Против."</string>
@@ -651,7 +651,7 @@
     <string name="spoken_emoji_1F449" msgid="4764447975177805991">"Белая рука с указательным пальцем вправо с тыльной стороны."</string>
     <string name="spoken_emoji_1F44A" msgid="7197417095486424841">"Кулак."</string>
     <string name="spoken_emoji_1F44B" msgid="1975968945250833117">"Машущая раскрытая ладонь."</string>
-    <string name="spoken_emoji_1F44C" msgid="3185919567897876562">"Пальцы, сложенные знаком О\'кей."</string>
+    <string name="spoken_emoji_1F44C" msgid="3185919567897876562">"Пальцы, сложенные знаком Окей."</string>
     <string name="spoken_emoji_1F44D" msgid="6182553970602667815">"Большой палец вверх."</string>
     <string name="spoken_emoji_1F44E" msgid="8030851867365111809">"Большой палец вниз."</string>
     <string name="spoken_emoji_1F44F" msgid="5148753662268213389">"Аплодисменты."</string>
diff --git a/java/res/values-ru/strings.xml b/java/res/values-ru/strings.xml
index 6dc70d7a5..40ab5b582 100644
--- a/java/res/values-ru/strings.xml
+++ b/java/res/values-ru/strings.xml
@@ -58,7 +58,7 @@
     <string name="use_contacts_dict" msgid="4435317977804180815">"Подсказывать имена"</string>
     <string name="use_contacts_dict_summary" msgid="6599983334507879959">"Подсказывать исправления на основе имен из списка контактов"</string>
     <string name="use_personalized_dicts" msgid="5167396352105467626">"Пользовательские словари"</string>
-    <string name="enable_metrics_logging" msgid="5506372337118822837">"Помочь улучшить <xliff:g id="APPLICATION_NAME">%s</xliff:g>"</string>
+    <string name="enable_metrics_logging" msgid="5506372337118822837">"Помочь улучшить приложение <xliff:g id="APPLICATION_NAME">%s</xliff:g>"</string>
     <string name="use_double_space_period" msgid="8781529969425082860">"Точки автоматически"</string>
     <string name="use_double_space_period_summary" msgid="6532892187247952799">"Вводить точку с пробелом двойным нажатием пробела."</string>
     <string name="auto_cap" msgid="1719746674854628252">"Заглавные автоматически"</string>
diff --git a/java/res/values-sw/strings.xml b/java/res/values-sw/strings.xml
index a7703fb43..16cfa80c5 100644
--- a/java/res/values-sw/strings.xml
+++ b/java/res/values-sw/strings.xml
@@ -52,9 +52,9 @@
     <string name="show_language_switch_key_summary" msgid="7343403647474265713">"Onyesha wakati lugha ingizo mbalimbali zinapowezeshwa"</string>
     <string name="key_preview_popup_dismiss_delay" msgid="6213164897443068248">"Kuchelewesha kutupa kitufe ibukizi"</string>
     <string name="key_preview_popup_dismiss_no_delay" msgid="2096123151571458064">"Hakuna kuchelewa"</string>
-    <string name="key_preview_popup_dismiss_default_delay" msgid="2166964333903906734">"Chaguo msingi"</string>
+    <string name="key_preview_popup_dismiss_default_delay" msgid="2166964333903906734">"Chaguomsingi"</string>
     <string name="abbreviation_unit_milliseconds" msgid="8700286094028323363">"Milisekunde <xliff:g id="MILLISECONDS">%s</xliff:g>"</string>
-    <string name="settings_system_default" msgid="6268225104743331821">"Chaguo msingi la mfumo"</string>
+    <string name="settings_system_default" msgid="6268225104743331821">"Chaguomsingi la mfumo"</string>
     <string name="use_contacts_dict" msgid="4435317977804180815">"Pendekeza majini ya Anwani"</string>
     <string name="use_contacts_dict_summary" msgid="6599983334507879959">"Tumia majina kutoka kwa Anwani kama mapendekezo na marekebisho"</string>
     <string name="use_personalized_dicts" msgid="5167396352105467626">"Mapendekezo yaliyobadilishwa kukufaa"</string>
@@ -139,7 +139,7 @@
     <string name="prefs_key_longpress_timeout_settings" msgid="6102240298932897873">"Ucheleweshaji unapobofya kitufe na kushikilia"</string>
     <string name="prefs_enable_emoji_alt_physical_key" msgid="5963640002335470112">"Emoji za kibodi halisi"</string>
     <string name="prefs_enable_emoji_alt_physical_key_summary" msgid="5259484820941627827">"Kitufe halisi cha Alt huonyesha kibao cha emoji"</string>
-    <string name="button_default" msgid="3988017840431881491">"Chaguo msingi"</string>
+    <string name="button_default" msgid="3988017840431881491">"Chaguomsingi"</string>
     <string name="setup_welcome_title" msgid="6112821709832031715">"Karibu kwenye <xliff:g id="APPLICATION_NAME">%s</xliff:g>"</string>
     <string name="setup_welcome_additional_description" msgid="8150252008545768953">"kwa Kuandika kwa ishara"</string>
     <string name="setup_start_action" msgid="8936036460897347708">"Anza kutumia"</string>
diff --git a/java/res/values-zh-rHK/strings.xml b/java/res/values-zh-rHK/strings.xml
index 951d4350d..f5b7c949d 100644
--- a/java/res/values-zh-rHK/strings.xml
+++ b/java/res/values-zh-rHK/strings.xml
@@ -77,9 +77,9 @@
     <string name="auto_correction_threshold_mode_aggressive" msgid="7319007299148899623">"加強模式"</string>
     <string name="auto_correction_threshold_mode_very_aggressive" msgid="1853309024129480416">"極度加強模式"</string>
     <string name="bigram_prediction" msgid="1084449187723948550">"建議下一個字詞"</string>
-    <string name="bigram_prediction_summary" msgid="3896362682751109677">"根據前一個字詞提出建議"</string>
+    <string name="bigram_prediction_summary" msgid="3896362682751109677">"根據上一個字詞提供建議"</string>
     <string name="gesture_input" msgid="826951152254563827">"啟用手勢輸入功能"</string>
-    <string name="gesture_input_summary" msgid="9180350639305731231">"透過滑動手指寫出字母來輸入字詞"</string>
+    <string name="gesture_input_summary" msgid="9180350639305731231">"透過在字母上滑動輸入字詞"</string>
     <string name="gesture_preview_trail" msgid="3802333369335722221">"顯示觸控蹤跡"</string>
     <string name="gesture_floating_preview_text" msgid="4443240334739381053">"動態浮動預覽"</string>
     <string name="gesture_floating_preview_text_summary" msgid="4472696213996203533">"在啟用手勢輸入時顯示建議的字詞"</string>
-- 
2.17.1

